mtp_api/apps/prison/migrations/0005_populate_prisoner_hash.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import hashlib
+
+from django.db import models, migrations
+
+
+def calculate_prisoner_hash(location):
+    original = '{number}_{dob}'.format(
+        number=location.prisoner_number.lower(),
+        dob=location.prisoner_dob.strftime('%d/%m/%Y')
+    )
+    hash_object = hashlib.sha256(original.encode())
+    return hash_object.hexdigest()
```

Comment:
I wonder if we should make this use `_calculate_prisoner_hash` from the model (and maybe make that `PrisonerLocation.generate_hash(number, dob)`)?